site/src/pages/TaskPage/TaskAppIframe.tsx

```diff
@@ -95,6 +93,15 @@ export const TaskAppIFrame: FC<TaskAppIFrameProps> = ({
 					className={"w-full h-full border-0"}
 					allow="clipboard-read; clipboard-write"
 				/>
+			) : app.health === "unhealthy" ? (
+				<div className="w-full h-full flex flex-col items-center justify-center">
+					<h3 className="m-0 font-medium text-content-primary text-base">
+						App unhealthy
+					</h3>
+					<span className="text-content-secondary text-sm">
+						Check the logs for details
+					</span>
```

Comment:
suggestion: add some more details so it's clear what the relevant app is, e.g.


```suggestion
						App "${app.name}" unhealthy
					</h3>
					<span className="text-content-secondary text-sm">
					Here are some troubleshooting steps you can take:
					<ul>
					  <li>Try running the following inside your workspce: <code>curl -v "${app.healthcheck.url}"</code></li>
						<li>Check <code>/tmp/coder-agent.log</code> inside your workspace "${workspace.name}" for more information.</li>
					</ul>
					</span>
```